junit-jupiter-api/junit-jupiter-api.gradle.kts

```diff
@@ -9,6 +9,7 @@ javaLibrary {
 }
 
 dependencies {
+	api("org.jetbrains.kotlinx:kotlinx-coroutines-core:${Versions.coroutines}")
```

Comment:
I wasn't aware this would require an additional dependency. Is there a way to avoid that, e.g. would `compileOnly` suffice?

Reply:
This should be `compileOnly` like the `kotlin-stdlib` below.